test/test_permeation_problem.py

```diff
@@ -134,4 +265,5 @@ def siverts_law(T, S_0, E_S, pressure):
 
 
 if __name__ == "__main__":
-    test_permeation_problem()
+    # test_permeation_problem()
+    test_permeation_problem_multi_volume()
```

Comment:
Can remove these lines altogether
```suggestion
    pass
```